### PR TITLE
Update script to set orderFulfillmentStatusTypeID

### PR DESCRIPTION
**Pull request by chriskent**

### CODE REVIEW

File: config/scripts/database/v4_0.cfm [hunk 1]
```diff
@@ -118,6 +118,31 @@ Notes:
...
+				SELECT CASE
+					WHEN min(oi.quantity) - sum(odi.quantity) = 0 THEN '159118d67de3418d9951fc629688e194'
+					WHEN min(oi.quantity) - sum(odi.quantity) > 0 THEN 'fefc92c1d8184017aa65cdc882bdf636'
+					ELSE 'b718b6fadf084bdaa01e47f5cc1a8265'
+				END
+				FROM SwOrderItem oi
+					LEFT JOIN SwOrderDeliveryItem odi ON odi.orderItemID = oi.orderItemID
+				WHERE oi.orderFulfillmentID = SwOrderFulfillment.orderFulfillmentID
+       		)
+	</cfquery>
```
Comment (sumitverma) [inline config/scripts/database/v4_0.cfm]: We need to make sure this query is compatible with mysql, sql server and oracle.

**Comment by gregmoser** [comment]
Also this PR should be made into 'feature' I believe.  Ryan can confirm

File: config/scripts/database/v4_0.cfm (hunk 1 above)
Comment (chriskent) [inline config/scripts/database/v4_0.cfm]: I have tested against MS SQL, do not have Oracle installed and no current V4 MySQL database available.

Comment (dzikakis) [inline config/scripts/database/v4_0.cfm]: I can confirm this query works on MySQL 5.5+

**Comment by rmarchandTen24** [comment]
Confirmed

**Pull request closed without merging**